app/todo_lists/resources.py

```diff
@@ -0,0 +1,123 @@
+from flask import request, session
+from flask_restful import Resource
+from sqlalchemy import and_
+
+from app import db, TodoList, TodoListPermission
+from app.lib import authentication, response_util, status
+
+
+class TodoListMultiResource(Resource):
+    """Class for creating and accessing lists.
+    """
+    @authentication.requiresAuth
+    def get(self):
+        """Method gets a list of logged in user's todoLists and returns them in
+        an Ok response.
+        """
+        userId = session.get('userId')
+        permissions = TodoListPermission.query.filter_by(userId=userId)
+        todoListIds = [permission.todoListId for permission in permissions]
+
+        if not todoListIds:
+            return response_util.buildOkResponse([])
+
+        todoLists = TodoList.query.filter(TodoList.id.in_(todoListIds))
+
+        return response_util.buildOkResponse([todoList.toDict()
+                                              for todoList in todoLists])
+
+    @authentication.requiresAuth
+    def post(self):
+        """Method creates a todoList and returns it in an Ok response.
+        """
+        if not(request.form.get('name')):
+            raise status.BadRequest()
+
+        todoList = TodoList(request.form.get('name'), session.get('userId'))
+        db.session.add(todoList)
+        db.session.commit()
+
+        permission = TodoListPermission(session.get('userId'), todoList.id)
+        db.session.add(permission)
+        db.session.commit()
+
+        return response_util.buildOkResponse(todoList.toDict())
+
+
+class TodoListResource(Resource):
+    """Class to get, update, or delete a single todoList.
+    """
+    @authentication.requiresAuth
+    def put(self, todoListId):
+        """Method that updates and returns a todoList in an Ok response.
+
+        Args:
+            listId - An integer, primary key that identifies the todo list.
+        """
+        userId = session.get('userId')
+        todoList = TodoList.query.get(todoListId)
+        permission = TodoList.query.filter()
+
+        TodoListPermission.query.filter_by(userId=userId)
+
+        if todoList is None:
+            raise status.NotFound()
+
+        permission = TodoListPermission.query.filter(
+                           and_(TodoListPermission.todoListId == todoList.id,
```

Comment:
Let's not indent this and_ so much. Usually we just indent once when continuing in a new line and not in an indented context. But if your linter is telling you otherwise, then no problem!